### PR TITLE
[INLONG-4259][Sort] Import inlong format json serialize/deserialize

### LINKED ISSUE #4259: [Feature][Sort] Import inlong format json serialize/deserialize
### Description

Import inlong format json serialize/deserialize

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by thexiay**
### Title Name: [INLONG-4259][Sort] Import inlong format json serialize/deserialize

Import inlong format json serialize/deserialize

Fixes #4259

### Motivation

Import inlong format json serialize/deserialize

### Modifications

Add Json Serialize InLongMsg format and its ut.

### CODE REVIEW

File: inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.protocol.node.format;
+
+import org.apache.inlong.sort.SerializeBaseTest;
+
+/**
+ * Test for {@link AvroFormat}
```
Comment (thexiay) [inline inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormatTest.java:23]: doc error.

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty(value = "innerFormat")
+    private Format innerFormat;
+
+    @JsonProperty(value = "ignoreParseErrors", defaultValue = "false")
+    private Boolean ignoreParseErrors;
+
+    @JsonCreator
+    public InLongMsgFormat(@JsonProperty(value = "innerFormat") Format innerFormat,
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java:48]: Please add some java comments.

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            @JsonProperty(value = "ignoreParseErrors", defaultValue = "false") Boolean ignoreParseErrors) {
+        this.innerFormat = innerFormat;
+        this.ignoreParseErrors = ignoreParseErrors;
+    }
+
+    @JsonCreator
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java:54]: Maybe it is must unique?

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    public InLongMsgFormat() {
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java:55]: Please add some java comments.

Comment (thexiay) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java:55]: already add in class define.

Comment (thexiay) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/InLongMsgFormat.java:55]: > 

already add in class define.

**Review by yunqingmoswu — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]
LGTM

**Review by healchow — APPROVED** [review]

**Pull request merged**